### PR TITLE
feat: disable curly rule

### PR DESCRIPTION
**Pull request by stefvw93**
Disable curly
- current setup disallows single line control flow; subjective, but annoying imo, I want to `if (statement) return`

### CODE REVIEW

**Review by thomasowow — COMMENTED** [review]
> JavaScript allows the omission of curly braces when a block contains only one statement. However, it is considered by many to be best practice to never omit curly braces around blocks, even when they are optional, because it can lead to bugs and reduces code clarity. 

Kind of agree with this, mostly for consistency. Inconsistency sometimes is less readable in my opinion.

**Comment by stefvw93** [comment]
@thomasowow I added multi-or-nest

Incorrect
```
/*eslint curly: ["error", "multi-or-nest"]*/

if (!foo)
    foo = {
        bar: baz,
        qux: foo
    };

while (true)
  if(foo)
      doSomething();
  else
      doSomethingElse();

if (foo) {
    foo++;
}

while (true) {
    doSomething();
}

for (var i = 0; foo; i++) {
    doSomething();
}
```

correct
```
/*eslint curly: ["error", "multi-or-nest"]*/

if (!foo) {
    foo = {
        bar: baz,
        qux: foo
    };
}

while (true) {
  if (foo) doSomething();
  else doSomethingElse();
}

if (foo) foo++;

while (true) doSomething();

for (var i = 0; foo; i++) doSomething();
```

**Review by Sjoertjuh — APPROVED** [review]

File: packages/eslint-config-basic/index.js [hunk 1]
```diff
@@ -353,11 +353,14 @@ module.exports = {
...
-                next: ['return', 'export', 'class', 'if', 'for'],
-            },
+            { blankLine: 'always', prev: 'multiline-expression', next: '*' },
+            { blankLine: 'always', prev: '*', next: 'multiline-expression' },
+            { blankLine: 'always', prev: 'multiline-block-like', next: '*' },
+            { blankLine: 'always', prev: '*', next: 'multiline-block-like' },
+            { blankLine: 'always', prev: 'multiline-const', next: '*' },
+            { blankLine: 'always', prev: '*', next: 'multiline-const' },
+            { blankLine: 'always', prev: 'multiline-let', next: '*' },
+            { blankLine: 'always', prev: '*', next: 'multiline-let' },
```
Comment (barisbora) [inline packages/eslint-config-basic/index.js:363]: Just question it covers return?

Comment (barisbora) [reply packages/eslint-config-basic/index.js:363]: Screenshot 2024-05-17 at 15 32 51

I think we should keep return at least, I also really like if and for too

Comment (stefvw93) [reply packages/eslint-config-basic/index.js:363]: Agree, though: Could you create a separate PR for this? 🙏🏻

**Pull request merged**